### ISSUE TITLE
Round conclusion event

### ISSUE THREAD

**Issue by blaberry** [NeotokyoRebuild/neo#217]
We might not need this? If we have the comp plugin as parity

**Comment by blaberry** [comment]
Let's abandon this due to having comp natively.

**Comment by Rainyan** [comment]
> Assuming its for plugins it might still be useful, but if not they can still hook in to the output on neo_game_config probably

I agree that having useful events for different, well, events, would be useful for plugins/server modding. It might be a good top-level issue with all the desired events, once we've decided what those would be.

Another good thing to have would be compatibility with [OG NT events](https://wiki.alliedmods.net/Neotokyo_Events) for better plugin compatibility. We currently have mock-ups of `game_round_start` and `game_round_end` ifdef'd away in the code base; those would probably be quite trivial to get working.
For `player_death` event, we're extending the OG NT values which is fine, but it seems ours is missing the original's `icon` field, which we could add for compat.

**Comment by Agiel** [comment]
As mentioned in discord, the mapchooser plugin also relies on this to show the vote menu when a team is close to winning. For OGNT we use a modified mapchooser, but I think it's better to make NT;RE work with the stock plugin. There's the option of submitting a PR to SourceMod, but I'd rather we conform to the "standard".

**Comment by blaberry** [comment]
Let's keep this issue for "native SM mapchooser works" and I'll make a separate issue for "Events we want".
